_kbarticles/2023-08-07-advantages-of-multiple-application-instances.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Scaling your cloud.gov applications"
+date: August 7, 2023
+excerpt: How and why to scale your cloud.gov applications by adding more application instances
+---
+
+## Importance of multiple application instances
+
+One of the benefits of Cloud.gov is the ability to [deploy applications with a simple command](https://cloud.gov/docs/deployment/deployment/#how-deployment-works). 
+
+Once an application is up and available, resiliency of that application becomes a consideration and as such running multiple application instances is paramount. Running multiple instances of an application increases the resilience and availability of the application.
+
+Multiple application instances allow customer traffic to continue being served if there are any issues with any individual application instance such as if an underlying platform component is being upgraded. Additionally having multiple application instances provides decreased chance of downtime. 
```

Comment:
```suggestion
```

Having multiple instances doesn't protect you from platform-level outages. Requests to all application instances have to go through the same routing infrastructure, so if any of that infrastructure failed, you would still have outages.